packages/esbuild/esbuild.bzl

```diff
@@ -168,6 +174,13 @@ See https://esbuild.github.io/api/#format for more details
             doc = """Link the workspace root to the bin_dir to support absolute requires like 'my_wksp/path/to/file'.
     If source files need to be required then they can be copied to the bin_dir with copy_to_bin.""",
         ),
+        "max_threads": attr.string(
```

Comment:
Any reason to not make this an `int`?

Reply:
Yeah I guess it could be, it's got to be converted to a string for `env` then, but it might make a nicer API. I don't have a strong opinion either way

Reply:
I figured converting int => int-string is better then simply sending "asdlfkj" through 🤷